hack/make/.binary

```diff
@@ -50,10 +50,14 @@ if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARC
 	esac
 fi
 
-# -buildmode=pie is not supported on Windows.
-if [ "$(go env GOOS)" != "windows" ]; then
-	BUILDFLAGS+=( "-buildmode=pie" )
-fi
+# -buildmode=pie is not supported on Windows and Linux on mips.
+case "$(go env GOOS),$(go env GOARCH)" in
```

Comment:
I think this would be more "Go standard" if we used `/` instead of `,`; for example, the output of `go tool dist list` is `os/arch`:

```console
$ go tool dist list | grep mips
linux/mips
linux/mips64
linux/mips64le
linux/mipsle
```